zoo_frontend/src/pages/user/user.jsx

```diff
@@ -2,14 +2,33 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import Link from 'next/link';
 import { Button } from '@material-ui/core';
+import MaterialTable from 'material-table';
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import Search from '@material-ui/icons/Search';
+import FirstPage from '@material-ui/icons/FirstPage';
+import LastPage from '@material-ui/icons/LastPage';
+import NextPage from '@material-ui/icons/ChevronRight';
+import PreviousPage from '@material-ui/icons/ChevronLeft';
+
+import UsersAPI from '../../static/Users';
 
 import { hasAccess, Admin, User } from '../PageAccess';
 
 class Home extends Component {
+  /**
+   * Server side data retrieval
+   */
+  static async getInitialProps({ authToken }) {
+    this.api = new UsersAPI(authToken);
```

Comment:
'this' should not be used in a static function

Reply:
Took out 'this'.